framework/src/modules/nft/types.ts

```diff
@@ -50,6 +50,8 @@ export interface TokenMethod {
 export interface NFTMethod {
 	getChainID(nftID: Buffer): Buffer;
 	destroy(methodContext: MethodContext, address: Buffer, nftID: Buffer): Promise<void>;
+	getNFTOwner(methodContext: ImmutableMethodContext, nftID: Buffer): Promise<Buffer>;
+	getLockingModule(methodContext: ImmutableMethodContext, nftID: Buffer): Promise<string>;
```

Comment:
I think the interface is needed to avoid circular dependency, as `InternalMethod` does not import `NFTMethod`, but the interface.

I will create a separate issue to refactor internal method so there is no circular dependency, and that the interface is not needed.